doc/releases/quincy.rst

```diff
@@ -10,6 +10,19 @@ v17.2.9 Quincy
 
 This is the ninth backport (hotfix) release in the Quincy series. We recommend that all users update to this release.
 
+Users should expect to see the el8 rpm subdirectory empty and the "dnf" commands are expected
+to fail with 17.2.9.
+They can choose to use 17.2.9 RPM packages for centos 8/el8 provided by CERN as a community
+member or continue to stay at 17.2.7 following instructions
+from https://docs.ceph.com/en/latest/install/get-packages/#rhel, the ceph.repo file should
+point to https://download.ceph.com/rpm-17.2.7/el8 instead of https://download.ceph.com/rpm-quincy/el8
+
+These CERN packages come with no warranty and have not been tested. The software in them has been
```

Comment:
Here we say they have not been tested, but then that they have.  The latter I think refers to container deployments but it might be good to make that more clear.

Reply:
Agree clarity would be of help.
I am unsure if the latter sentence refers exclusively to container deployments or more generally to the codebase.